### ISSUE TITLE
Image cannot be load when edit the colleciton

### ISSUE THREAD

**Issue by rathahin** [VeliovGroup/meteor-autoform-file#16]
Story:
- I have the Advertisements collection. There is a field called "AdvertiseImage" which using autoForm file to upload an image.

Issue:
- It is working fine on insert new document.
- On edit created document, the image show fileUpload box instead of uploaded image.

Investigation:
- The id which giving to FileCollection.findOne({id}) is empty
- On this break point
ReactiveVar get empty 
![image](https://cloud.githubusercontent.com/assets/4031452/21425197/c92d0898-c858-11e6-82a8-a8fc53eba3e6.png)

**Comment by dr-dimitru** [comment]
Hello @rathahin ,

Could you replicate same behaviour by running [demo app](https://github.com/VeliovGroup/files-gridfs-autoform-example)?

**Comment by rathahin** [comment]
Thank you for your reply!

The demo app is working fine. but the difference is that I use autoForm which generate by meteor-admin.

**Comment by dr-dimitru** [comment]
@rathahin take a look on [this issue](https://github.com/VeliovGroup/meteor-autoform-file/issues/12) related to meteor-admin too.

For some reason most of the developers faced this issue. Could you please create minimum app/settings where I can reproduce this behaviour?

**Comment by rathahin** [comment]
Hi @dr-dimitru , Thank again for your reply. I have create project repo https://github.com/rathahin/autoform-files-case . it show the issue when upload then edit the collection, the image doesn't display. Instead of display the file input to select image again.

**Comment by rathahin** [comment]
Hello

In the function Template.afFileUpload.onCreated(function () { ... } the this.data.value is given as empty string.

**Comment by dr-dimitru** [comment]
Hi @rathahin ,

I checked the code, can't find the reason for this behaviour.
I'll be back later, and wish be back with some results.

**Comment by rathahin** [comment]
Dear @dr-dimitru , I modified the fileUpload.js:

```javascript
Template.afFileUpload.helpers({
  previewTemplate: function () {
    return Template.instance().previewTemplate;
  },
  uploadTemplate: function () {
    return Template.instance().uploadTemplate;
  },
  currentUpload: function () {
    return Template.instance().currentUpload.get();
  },
  fileId: function () {
    return Template.instance().fileId.get() || this.value;
  },
  uploadedFile: function () {
    var id = Template.instance().fileId.get() || this.value;
    return global[Template.instance().collectionName()].findOne({
      _id: id
    });
  }
});
```

And it seem to be work for my case, both insert and update can now display the image. but I don't really now this is proper solution. I have commit code to this repo: https://github.com/rathahin/autoform-files-case

**Comment by dr-dimitru** [comment]
Hello @rathahin ,

Thank you for contribution.
Your suggestions is published in `v1.0.10`
